libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/RouteComparator.java

```diff
@@ -0,0 +1,73 @@
+package com.mapbox.services.android.navigation.ui.v5;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.mapbox.api.directions.v5.models.DirectionsResponse;
+import com.mapbox.api.directions.v5.models.DirectionsRoute;
+import com.mapbox.api.directions.v5.models.RouteLeg;
+
+import java.util.List;
+
+class RouteComparator {
+
+  private static final int FIRST_ROUTE = 0;
+  private static final int ONE_ROUTE = 1;
```

Comment:
adjust naming for consistency

Reply:
Consistent with what exactly? Sorry not following the comment

Reply:
Just understood what that meant, sorry I thought they were both referring to indexes. Maybe `SIZE_ONE_ROUTE` would be more explicit for the second field?